sketch/hash_base.hpp

```diff
@@ -74,9 +81,14 @@ class HashBase : public SketchBase<std::vector<T>, true> {
                 return val;
             }
             case HashAlgorithm::crc32: {
-                T val = _mm_crc32_u32((uint32_t)crc32_base, (uint32_t)key);
+                uint32_t val = _mm_crc32_u32(hash_seed, (uint32_t)key);
                 return _mm_crc32_u32((uint32_t)val, (uint32_t)index);
             }
+            case HashAlgorithm::murmur:
+                uint32_t val;
+                MurmurHash3_x86_32(&key, sizeof(key), hash_seed, &val);
```

Comment:
Here is what MASH does:
https://github.com/marbl/Mash/blob/master/src/mash/hash.cpp#L14-L24

Seems like they use one call to the `_128` variant on 64bit architectures, and then only use half of the output bits.

Maybe `key` and `index` can also be combined into 1 to only do one call?

Reply:
I tried to emulate what they do, and also called crc32 twice with two different seeds for 64 bit types.